src/components/pages/dashboard/card/odk-modal/index.tsx

```diff
@@ -0,0 +1,117 @@
+import { WarningIcon } from "@chakra-ui/icons";
+import {
+  Box,
+  Button,
+  Heading,
+  Link,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  Stat,
+  StatHelpText,
+  useDisclosure,
+} from "@chakra-ui/react";
+import useGlobalState from "@hooks/use-global-state";
+import { ENDPOINT } from "@static/constants";
+import useTranslation from "next-translate/useTranslation";
+import React from "react";
+
+import AppUserQrModal from "./qr-modal";
+
+function SeeQrModal({ index, item, user }) {
+  const { t } = useTranslation();
+
+  const { isOpen: isQrOpen, onOpen: onQrOpen, onClose: onQrClose } = useDisclosure();
+
+  return (
+    <tr key={index}>
+      <td>
+        <Box userSelect="all" fontSize="sm" className="elipsis">
+          {item.name}
+        </Box>
+      </td>
+      <td>
+        <Button variant="link" onClick={onQrOpen} leftIcon={<WarningIcon />}>
+          {t("common:action.see_code")}
+        </Button>
+      </td>
+      <AppUserQrModal
+        key={index}
+        projectName={item.name}
+        qrUrl={`${ENDPOINT.ODK}/app-user/qr-code/${user.userName}-suser${user.id}/${item.id}`}
+        isQrOpen={isQrOpen}
+        onQrClose={onQrClose}
+      />
+    </tr>
+  );
+}
+
+export default function OdkModal({ isOpen, onClose, odkLink }) {
+  const { t } = useTranslation();
+
+  const { user, isOdkWebUser, userAppProjectList } = useGlobalState();
+
+  return (
+    <>
+      <Modal isOpen={isOpen} onClose={onClose}>
+        <ModalOverlay />
+        <ModalContent>
+          <ModalHeader></ModalHeader>
+          <ModalCloseButton />
+          <ModalBody>
+            {isOdkWebUser && (
+              <Heading size="sm" mb={3}>
+                <b>ODK webuser</b>
```

Comment:
1) put this in translation file instead of hardcoding it.
2) can you bold it without using this jsx tag? like through fontWeight props in chakra ui elements itself